mustermann/lib/mustermann/ast/translator.rb

```diff
@@ -73,7 +82,6 @@ def self.translate(*types, &block)
         Class.new(const_get(:NodeTranslator)) do
           register(*types)
           define_method(:translate, &block)
-          ruby2_keywords :translate
```

Comment:
My *guess* is this one is producing the warning, and it should probably be `define_method(:translate, &block.ruby2_keywords)` instead.

Reply:
Still seeing the warnings with `define_method(:translate, &block.ruby2_keywords)`

Reply:
Could you post the full warning output (it should show where it happens), and how to reproduce it?

Reply:
`
.../mustermann/mustermann/lib/mustermann/ast/translator.rb:75: warning: Skipping set of ruby2_keywords flag for proc (proc accepts keywords or proc does not accept argument splat)
`